openslides_backend/action/action.py

```diff
@@ -107,13 +108,7 @@ def perform(
             self.validate_instance(instance)
             # perform permission check not for internal actions
             if not internal:
-                try:
-                    self.check_permissions(instance)
-                except PermissionDenied as e:
-                    if e.message.startswith("Missing"):
-                        msg = f"You are not allowed to perform action {self.name}."
-                        e.message = msg + " " + e.message
-                    raise e
```

Comment:
You can keep this and add the action name here, this way you don't have to pass `self.name` for every exception. You only need to catch `MissingPermission` then.

Reply:
I dislike to update the message here, if it is not necessary.

Reply:
The whole point of this PR was to remove the duplication from the permission exceptions, and that would do exactly that.

Reply:
The code is in again with the `MissingPermission` and without the `if e.message.startswith("Missing"):`